### ISSUE TITLE
Add VIP through DHCP

### ISSUE THREAD

**Issue by thebsdbox** [plunder-app/kube-vip#53]
Investigate using DHCP to add a VIP address 

cc / @mylesagray

**Comment by thebsdbox** [comment]
Time to turn this into GO

```
sudo ip link add mymacvlan1 link  ens192 type macvlan mode bridge
sudo ifconfig mymacvlan1 up
sudo dhclient mymacvlan1
ip addr show mymacvlan1

8: mymacvlan1@ens192: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN group default qlen 1
    link/ether ca:aa:6f:4a:bc:2c brd ff:ff:ff:ff:ff:ff
    inet 192.168.0.173/24 brd 192.168.0.255 scope global mymacvlan1
       valid_lft forever preferred_lft forever
    inet6 fe80::c8aa:6fff:fe4a:bc2c/64 scope link 
       valid_lft forever preferred_lft forever
```

**Comment by thebsdbox** [comment]
This is being tacked in #55

**Comment by plunder-app** [comment]
Closed with #55
